Dockerfile

```diff
@@ -1,75 +1,63 @@
-FROM ubuntu:20.04
+FROM ubuntu:24.04
 
 ENV WORKSPACE_DIR /root
-ENV FIRMWARE_DIR ${WORKSPACE_DIR}/Firmware
+ENV FIRMWARE_DIR ${WORKSPACE_DIR}/px4
 ENV SITL_RTSP_PROXY ${WORKSPACE_DIR}/sitl_rtsp_proxy
 
 ENV DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true
 ENV DISPLAY :99
 ENV LANG C.UTF-8
 
 RUN apt-get update && \
-    apt-get install -y wget lsb-release && \
-    apt-get -y autoremove && \
-    apt-get clean autoclean && \
-    rm -rf /var/lib/apt/lists/{apt,dpkg,cache,log} /tmp/* /var/tmp/*
-
-RUN wget https://packages.osrfoundation.org/gazebo.gpg -O /usr/share/keyrings/pkgs-osrf-archive-keyring.gpg && \
-    echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/pkgs-osrf-archive-keyring.gpg] http://packages.osrfoundation.org/gazebo/ubuntu-stable $(lsb_release -cs) main" | tee /etc/apt/sources.list.d/gazebo-stable.list > /dev/null
-
-
-RUN apt-get update && \
-    apt-get install -y bc \
+    apt-get install -y \
                        cmake \
-                       gazebo11 \
+                       curl \
                        git \
+                       gnupg \
                        gstreamer1.0-plugins-bad \
                        gstreamer1.0-plugins-base \
                        gstreamer1.0-plugins-good \
                        gstreamer1.0-plugins-ugly \
-                       iproute2 \
```

Comment:
I think you still need the `ip route` command for

https://github.com/JonasVautherin/px4-gazebo-headless/blob/ff7947026a0f350f4ba467fd050c05e91051f6d2/edit_rcS.bash#L27